net/src/icmp4/mod.rs

```diff
@@ -7,6 +7,10 @@ use crate::parse::{DeParse, DeParseError, LengthError, Parse, ParseError, ParseP
 use etherparse::Icmpv4Header;
 use std::num::NonZero;
 
+#[allow(unused_imports)] // re-export
```

Comment:
Why do you need this, given that this is seemingly unnecessary for ICMPv6 generation?

Reply:
Why not having it for ICMPv6 then?